k-distribution/tests/regression-new/cell-bag-sort-llvm/test.k.out

```diff
@@ -0,0 +1,7 @@
+[Error] Compiler: Cell bags are not supported except on the Java backend. If
+you want this feature, file an issue at https://github.com/kframework/k  As a
+workaround, you can add the attribute type="Set" and add a unique identifier to
+each element in the set.
```

Comment:
Why is this `Java` backend? Isn't this test run with LLVM backend?

Reply:
Except on.

Reply:
Ah, sorry I misread. Ok then, I think we should:

1.  Change the sentence which says `If you want this feature, file an issue ...` to have a period at the end.
2.  Instead link them directly to this issue: https://github.com/kframework/k/issues/1419 and tell them to comment on it instead of filing a new issue.

Reply:
Also, I would say instead "Cell bags are only supported on the Java backend." It's a bit more natural wording.